### ISSUE TITLE
[intelliJ][RunConf]Do not support accessability on the window of Run/Debug configuration

### ISSUE THREAD

**Issue by v-jiche** [microsoft/azure-tools-for-java#2855]
Build:
azure-toolkit-for-intellij-2018.3.develop.1014.03-05-2019

Repro Steps:
1. Open Run/Debug Configurations.
2. Click **Locally Run**.
![image](https://user-images.githubusercontent.com/35950097/53861739-0182ef00-4020-11e9-9cd5-2da8a5805367.png)
3. Tab away.
Result:
Can not tab to the part of Remotely Run. It is violated with MAS 07 and MAS 36
4. Click "**Remotely Run**".
5. Tab away.
Result:
It generate dead loops in the part of Job Configurations. It is violated with MAS 07.
![image](https://user-images.githubusercontent.com/35950097/53862233-a94cec80-4021-11e9-8f67-c5248f11bea2.png)

**Comment by t-rufang** [comment]
Replace the table with environment variable table.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1748129
